src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -87,7 +87,7 @@ public async Task<JObject> GetBuildConfig(SourceInfo<string> name, string reposi
                 ["xrefHostName"] = GetXrefHostName(docset.site_name, branch),
                 ["localization"] = new JObject
                 {
-                    ["defaultLocale"] = GetDefaultLocale(docset.site_name),
+                    ["defaultLocale"] = docset.locale,
```

Comment:
this will overwrite all loc builds' defaultLocale.
Say we build 'azure-docs-pr.ja-jp' with defaultLocale 'ja-jp'

Reply:
you are right